src/xenia/base/bit_field.h

```diff
@@ -29,7 +34,7 @@ struct bf {
   // For enum values, we strip them down to an underlying type.
   typedef
       typename std::conditional<std::is_enum<T>::value, std::underlying_type<T>,
-                                std::identity<T>>::type::type value_type;
+                                xe::identity<T>>::type::type value_type;
```

Comment:
Yeah LLVM's libc++ and GCC's libstdc++ do not have std::identity as it was never finalised in a standard, but MS kept it in their c++ std library...

I'll look into std::forward.